cloudevents/sdk/event/attributes/__init__.py

```diff
@@ -0,0 +1,13 @@
+#  Copyright 2018-Present The CloudEvents Authors
```

Comment:
Module names are usually singulars. I.e. in this case it should be `attribute`.

Also, I doubt whether such separation is worth having. That's a lot of file for a tiny amount of code.